taxi/views.py

```diff
@@ -87,3 +89,28 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
I know that's not your code, but when you have some task, you should try to make a glance on other parts beforehand. So change it also, accordingly to checklist, please.